runtime/lua/vim/_defaults.lua

```diff
@@ -169,6 +171,35 @@ do
       vim.lsp.buf.signature_help()
     end, { desc = 'vim.lsp.buf.signature_help()' })
   end
+
+  --- Map [d and ]d to move to the previous/next diagnostic. Map <C-W>d to open a floating window
+  --- for the diagnostic under the cursor.
+  ---
+  --- See |[d-default|, |]d-default|, and |CTRL-W_d-default|.
+  do
+    vim.keymap.set('n', ']d', function()
+      vim.diagnostic.goto_next({ float = false })
+    end, {
+      desc = 'Jump to the next diagnostic with the highest severity',
+    })
+
+    vim.keymap.set('n', '[d', function()
+      vim.diagnostic.goto_prev({ float = false })
+    end, {
+      desc = 'Jump to the previous diagnostic with the highest severity',
+    })
+
+    vim.keymap.set('n', '<C-W>d', function()
+      vim.diagnostic.open_float({ border = 'rounded' })
```

Comment:
Is there a reason border is hardcoded here? Ideally it would use the default or pick up value from what was provided to `vim.diagnostic.config()`

Reply:
Good catch! If you want to open a PR to change it I can review it.

Reply:
I opened a PR #28542 to remove the hardcoded border